frontend/src/pages/projects/notebook/NotebookStatusText.scss

```diff
@@ -3,11 +3,11 @@
 .odh-notebook-status-popover {
   &__starting-text {
     cursor: pointer;
-    text-decoration: underline var(--pf-v6-global--BorderWidth--sm) dashed
-      var(--pf-v6-global--BorderColor--200);
+    text-decoration: underline var(--pf-t--global--text-decoration--width--default) dashed
+      var(--pf-t--global--border--color--nonstatus--gray--default);
```

Comment:
If meant to mimic help text, could remove these color blocks

For hover below, could set text dec color to revert or text color regular